### ISSUE TITLE
Chat or Instruct model

### ISSUE THREAD

**Issue by arch-btw** [RUC-GSAI/YuLan-Mini#1]
Hi,

Congratulations on the release of your model!
I am curious to know if there there will be a chat or instruct version of this model?
Thank you

**Comment by huyiwen** [comment]
Of course. Our machines are hard at work training the Instruct version, and it’ll be released within the next couple of weeks.

**Comment by arch-btw** [comment]
That's great, thank you!

**Comment by huyiwen** [comment]
Update: https://huggingface.co/yulan-team/YuLan-Mini-Instruct-V1
